Resources/Prototypes/Maps/shoukou.yml

```diff
@@ -66,4 +66,3 @@
             SalvageSpecialist: [ 2, 4 ]
             MailCarrier: [ 1, 2 ]
             CargoTechnician: [ 2, 3 ]
-
```

Comment:
>:(

Reply:
Active war on newlines at the end of yml files. Invasion begun in shoukou.yml (I'll readd the newline)